staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -685,7 +685,7 @@ func (w *watchCache) isIndexValidLocked(index int) bool {
 // getAllEventsSinceLocked returns a watchCacheInterval that can be used to
 // retrieve events since a certain resourceVersion. This function assumes to
 // be called under the watchCache lock.
```

Comment:
document the meaning of the returned RV as well and how it should be used. potential for off-by-one errors is high here

Reply:
added comment - PTAL